### PR TITLE
Prettier again

### PR DESCRIPTION
**Pull request by kristojorg**
Ok so I did something wrong when rebasing my other prettier PR. I believe it had something to do with the `package-lock.json`. I kept getting type errors that didn't make any sense since the PR was only meant to update formatting. Anyways, it was due to the rebase so I just scrapped it and reapplied changes to a fresh fork off `master`, which is here. 

Again, let me know what other branches are active and I'll make commits in each doing essentially the same thing so that we can avoid intense merge conflicts later down the road.

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
I am okay with this. If the 80-char limit presents a problem, I think we can later update it. From your other PR comment, it's good to have `//prettier-ignore` available if we want to make prettier update something. There's currently no other branches for the `opds-web-client` right now.

**Comment by kristojorg** [comment]
Should we delete the other branches?

**Comment by EdwinGuzman** [comment]
Do you mean the other `prettier` branches? Yes, if they're closed they should, and this branch/PR is the most up to date.

**Comment by kristojorg** [comment]
Ok I deleted all the old prettier-related branches, but I was referring to the ones listed here:

image

I will keep `logic-components`, since I'm still using it, but should the others be deleted for organization sake if they're no longer in use?

**Comment by EdwinGuzman** [comment]
Yes, it's fine to delete, just not `gh-pages`.

**Comment by kristojorg** [comment]
lgtm?

**Comment by EdwinGuzman** [comment]
Ha, yes, that's what it stands for. We both approved so you can merge this in, but can you first update the version (only in `/packages/opds-web-client`) and update the changelog with this update? It should be 3.0.1 now.

**Comment by kristojorg** [comment]
I'll update the changelog on this branch, but I don't have permission to merge : /

File: packages/opds-web-client/CHANGELOG.md [hunk 1]
```diff
@@ -1,60 +1,85 @@
 ## Changelog
 
+### v0.3.1
+
+- added "prettier" code formatter and a git hook to run prettier before each commit
+- Updated tslint config to include prettier so that formatting is no longer a lint concern
+- ran prettier on all files, so many small file changes.
+
 ### v0.3.0
+
```
Comment (EdwinGuzman) [inline packages/opds-web-client/CHANGELOG.md]: Did prettier add these or did you?

**Comment by EdwinGuzman** [comment]
Thanks. Merging.

File: packages/opds-web-client/CHANGELOG.md (hunk 1 above)
Comment (kristojorg) [reply packages/opds-web-client/CHANGELOG.md]: Prettier did it. We can disable prettier for markdown files if you like

**Pull request merged**